helm/prometheus-rules/templates/platform/shield/alerting-rules/teleport.rules.yml

```diff
@@ -10,6 +10,24 @@ spec:
   groups:
   - name: teleport
     rules:
+    ## This alert pages if instances of the teleport-kube-agent statefulset stay not ready.
+    - alert: TeleportKubeAgentInstancesNotReady
+      annotations:
+        description: '{{`At least one teleport-kube-agent instance is unhealthy.`}}'
+        summary: Teleport Kube Agent has had one or more unhealthy instances for an extended period.
+        ## TODO: Add an ops recipe for known TKA recovery (e.g. deleting the pod to update the Secret).
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/teleport/
```

Comment:
@ssyno is there already any documentation about how to fix TKA in cases like the upgrade the other day? It would be good to write down the flow we had to do from updating the management cluster CMs, restarting teleport-operator, through to rolling the WC pods to pick up the new Secret